app/controllers/ctc/portal/portal_controller.rb

```diff
@@ -78,4 +75,16 @@ def ensure_current_submission
       redirect_to(action: :home) and return
     end
   end
+
+  def load_current_submission
+    @submission = current_client.efile_submissions.last
```

Comment:
this ensures that we dont have to load ALL the submissions then narrow it down to the last one -- does a SQL query to just get the last one from the db or returns nil